meetings/wasmtime/2022/wasmtime-01-06.md

```diff
@@ -0,0 +1,20 @@
+# January 6th Wasmtime project call
+
+**See the [instructions](../README.md) for details on how to attend**
+
+## Agenda
+1. Opening, welcome and roll call
+    1. Note: meeting notes linked in the invite.
+    1. Please help add your name to the meeting notes.
+    1. Please help take notes.
+    1. Thanks!
+1. Announcements
+    1. _Sumbit a PR to add your announcement here_
```

Comment:
s/Sumbit/Submit/ (pre-existing in all the minutes; missed by everyone until bnjbvr recently)